src/idas/idas_io.c

```diff
@@ -78,6 +78,25 @@ int IDASetUserData(void* ida_mem, void* user_data)
 
 /*-----------------------------------------------------------------*/
 
+int IDASetOwnUserData(void* ida_mem, sunbooleantype own_user_data)
```

Comment:
IDA is inconsistent about this but to follow the general convention prefix with `ida`
```suggestion
int idaSetOwnUserData(void* ida_mem, sunbooleantype own_user_data)
```